app/models/form.rb

```diff
@@ -38,9 +43,10 @@ def initialize(form_path, start_year = "", sections_in_form = [], type = "lettin
       @questions = pages.flat_map(&:questions)
       @start_date = Time.iso8601(form_definition["start_date"])
       @end_date = Time.iso8601(form_definition["end_date"])
+      @submission_deadline = Time.zone.local(2023, 6, 9)
```

Comment:
Surely this needs to be varied by year? OR is this hard-coded because from now on it could only ever apply to a 22/23 log?

Reply:
I think it drops into this part of conditional statement only for forms with a start year before 2022, submission_deadline above depends on the year